docs/docs/tutorial-basics/boundaries.md

```diff
@@ -6,7 +6,23 @@ sidebar_position: 3
 
 > Key concepts: Clamping, animations (handlers)
 
+## Resources
+
+- [Clamp](https://docs.swmansion.com/react-native-reanimated/docs/utilities/clamp/)
+
+## Challenge
+
+→ We need to keep him inside the screen boundaries.
+
+→ We want to drop him and make him bounce on the ground.
+
 <video controls autoplay loop style={{maxHeight: 640, minHeight: 200}}>
 
   <source src="https://user-images.githubusercontent.com/80724668/187310576-f6cf041b-3428-4989-94b2-c34346e66945.mov" />
 </video>
+
+💡 TIPS && HINTS:
+
+- You already have the gesture handler you need for this, consider using callbacks.
+- The library provides us a [clamp](https://docs.swmansion.com/react-native-reanimated/docs/utilities/clamp) function to bound the movement.
+- You can explore the withTiming config for the bouncing.
```

Comment:
Perhaps we can put backticks on the function names so they're clearly formatter (i.e. `withTiming`, `runOnJS`, etc)

![image](https://github.com/carozo/slimy/assets/3135138/a7aef625-5a1d-4c81-b4f1-9e70dde24929)